plugin/writer/icebergwriter/src/main/java/com/wgzhao/addax/plugin/writer/icebergwriter/IcebergWriter.java

```diff
@@ -59,7 +59,12 @@
 
 import java.io.IOException;
 import java.io.UncheckedIOException;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
 
 public class IcebergWriter
```

Comment:
which or whose errors are fixed ?
I cannot figure out according the changed codes.